gr-iio/lib/pluto_utils_v0.cc

```diff
@@ -9,10 +9,14 @@
 
 #include <gnuradio/iio/pluto_utils.h>
 #include <gnuradio/logger.h>
+#include <stdexcept>
 
+#ifdef LIBIIO_V1
+#include <iio/iio.h>
+#else
 #include <iio.h>
+#endif
```

Comment:
No sense in doing that here if you're only using that in the non-`LIBIIO_V1` case.